api/src/main/java/org/openmrs/ConceptMapType.java

```diff
@@ -11,18 +11,31 @@
 
 import org.codehaus.jackson.annotate.JsonIgnore;
 
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
 /**
  * ConceptMapType are used to define relationships between concepts and concept reference terms e.g
  * IS_A or SAME_AS, BROADER_THAN
  *
  * @since 1.9
  */
+@Entity
+@Table(name = "concept_map_type")
 public class ConceptMapType extends BaseChangeableOpenmrsMetadata {
```

Comment:
have you cross checked that all the attributes that you never included in this file do exist under the extended class

Reply:
yes @HerbertYiga